View/DataContexts/ArtistDataContext.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Net.Mime;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Windows.Controls;
+using System.Windows.Media.Imaging;
+using Controller.DbControllers;
+using Microsoft.Win32;
+using Model.Annotations;
+using Model.Database.Contexts;
+using Model.DbModels;
+using Soundify;
+
+namespace View.DataContexts
+{
+    public class ArtistDataContext : INotifyPropertyChanged
+    {
+        private static ArtistDataContext _instance;
+        public static ArtistDataContext Instance => _instance ??= new ArtistDataContext();
+
+        public TagLib.File SelectedSong { get; set; }
```

Comment:
Wat is dit?

Reply:
het bestand dat de gebruiker wil gaan uploaden

Reply:
Aaah, en dat kan alleen maar met TagLib?

Reply:
Kan vast ook wel met andere libraries, maar ik heb TagLib he meest voorbij zien komen